scripts/flake_checker.py

```diff
@@ -37,18 +37,19 @@
         'env': {
             'identifier': 'CIRCLECI',
             'user_info': 'CIRCLE_USERNAME',
-            'build_url': 'CIRCLE_BUILD_URL',
-            'build_id': None
-        }
+            'branch': 'CIRCLE_BRANCH',
+            'template_vars': ['CIRCLE_BUILD_URL']
+        },
+        'build_url_template': '%s',
     },
     'githubActions': {
         'env': {
             'identifier': 'GITHUB_ACTIONS',
             'user_info': 'GITHUB_ACTOR',
-            'build_url': None,
-            'build_id': 'GITHUB_RUN_ID',
+            'branch': 'GITHUB_REF',
+            'template_vars': ['GITHUB_REPOSITORY', 'GITHUB_RUN_ID'],
```

Comment:
Maybebuild_url_template_vars`? (or something around that?) [OR have `build_id` and `repo` as a separate key-value pair?]

Reply:
I like `build_url_template_vars`. I don't see the advantage to storing the template variables as key-value pairs though. Why do you think that would be helpful?

Reply:
I've updated the key to `build_url_template_vars`